api/src/main/java/com/cloud/offering/ServiceOffering.java

```diff
@@ -76,7 +76,7 @@
     /**
      * @return Does this service plan offer HA?
      */
-    boolean getOfferHA();
+    boolean isOfferHA();
```

Comment:
the java doc implies it should be called doesOfferHA()

Reply:
@DaanHoogland I would prefer to only have methods starting with either get or is.
Any other method name in a VO would not be usable for filtering,
according to the implementation of the VO MethodInterceptor

Reply:
good point @fmaximus so the generic dao breaks if we go any other way is your message, is it?

Reply:
Do you know the name of the class that does this interception, so we can inspect it?
Is it using method names or POJO's properties directly via their annotation?